### DISCUSSION TITLE
keyshade X FOSS Hack'24

### DISCUSSION THREAD

**Discussion by rajdip-b** [keyshade-xyz/keyshade#284] [Announcements]
Greetings from team Keyshade! We are glad to announce that we have been officially accepted as a Community FOSS Project at **FOSS Hack** this season. This helps us massively in community outreach, and helps our entire contributor community as well. 

## What can you do during this event?

We are maintaining a list of eligible issues labeled as `foss hack`, so do hop right in! Right now, we are just two developers maintaining the issues and the repository, so do let us know about your queries over GitHub discussions, Discord issue board, or even @ us: @rajdip-b @kriptonian1 

## Getting started

 Here's a list of links that might help you get started:

- **Register with FOSS Hack**: https://fossunited.org/fosshack/2024
- **Check out the enlisted issues**:  https://github.com/keyshade-xyz/keyshade/labels/foss%20hack
- **Join our Discord to have a chat with us**: https://discord.gg/kjP5PBWM

**NOTE:** Please go through the [contribution guidelines](https://github.com/keyshade-xyz/keyshade/blob/develop/CONTRIBUTING.md) before getting started, as that helps us massively in reviewing your PRs and getting them merged ASAP!

**Comment by tusharshah21** [comment]
hey @rajdip-b but the discord link is not working from my side could you please confirm it !
I would like to contribute

**Reply by rajdip-b** [reply]
Hey! Could you please try this link: https://discord.com/invite/dh8F3Dzt

**Reply by tusharshah21** [reply]
thanks
